src/Specification/Paths.php

```diff
@@ -35,6 +35,8 @@ public function getPaths()
      */
     public function getPathItemFor($path)
```

Comment:
- For micro-optimisation: isset is a bit faster than array_key_exists.
- This shoud explicitly return null if the path does not exist.
- The return annotation should reflect the null return value.

Reply:
Done :+1: